backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/FeaturedLibraries.kt

```diff
@@ -163,6 +163,7 @@ private fun getFeaturedLibraries(
 ) : List<KonanLibrary> {
     val remainingFeaturedLibraries = featuredLibraryFiles.toMutableSet()
     val result = mutableListOf<KonanLibrary>()
+    @Suppress("UNCHECKED_CAST")
```

Comment:
Hmm, can't we fix the signature?

Reply:
no, but we can do the following:
 val libraries = resolvedLibraries.getFullList(null).filterIsInstance<KonanLibrary>()